app/routes/vet/milk-test-bvd.js

```diff
@@ -0,0 +1,47 @@
+const Joi = require('joi')
+const { vetVisitData: { milkTestBvdResult } } = require('../../session/keys')
+const session = require('../../session')
+const getYesNoInvestigateRadios = require('../helpers/yes-no-investigate-radios')
+
+const id = 'milkTestBvdResult'
```

Comment:
Same as above, can this be removed and the usages replaced with the `milkTestBvdResult` session key please.